cmf-cli/Commands/publish/PublishCommand.cs

```diff
@@ -0,0 +1,222 @@
+using Cmf.Common.Cli.Attributes;
+using Cmf.Common.Cli.Constants;
+using Cmf.Common.Cli.Factories;
+using Cmf.Common.Cli.Interfaces;
+using Cmf.Common.Cli.Objects;
+using Cmf.Common.Cli.Utilities;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Cmf.Common.Cli.Commands
+{
+    /// <summary>
+    ///
+    /// </summary>
+    /// <seealso cref="Cmf.Common.Cli.Commands.BaseCommand" />
+    [CmfCommand("publish")]
+    public class PublishCommand : BaseCommand
+    {
+        #region Private Methods
+
+        /// <summary>
+        /// Publish Dependencies from one package. recursive operation
+        /// </summary>
+        /// <param name="outputDir">Destination for the dependencies package and also used for the current package</param>
+        /// <param name="repoUri">Source Location where the package dependencies should be downloaded</param>
+        /// <param name="packageId">Source Package ID</param>
+        /// <param name="packageVersion">Source Package Version</param>
+        /// <param name="loadedPackages">List of packages already processed.</param>
+        /// <param name="publishTests">True to publish test packages</param>
+        private void PublishDependenciesFromPackage(DirectoryInfo outputDir, Uri repoUri, string packageId, string packageVersion, List<string> loadedPackages, bool publishTests)
+        {
+            loadedPackages ??= new List<string>();
+            string dependencyFileName = $"{packageId}.{packageVersion}.zip";
```

Comment:
Shouldn't this be called `packageFileName`? We are not dealing with dependencies here.

Reply:
Fixed